GA-worksheet.md

```diff
@@ -6,18 +6,13 @@
 
 
 ##Project #2 Part #1: Header HTML
-1. What is the purpose of the header tag?
+1. What is the purpose of the header tag?Upper Part or a theme or like a title
```

Comment:
More specifically, the header tag holds the introductory material for the website, such as a logo and navigational links. Check it out on Buzzfeed.com. The top of the page that lets you know that you're on Buzzfeed is the header (inspect element to clarify).